README.md

```diff
@@ -289,35 +289,17 @@ To test the download route, the url is:
 
 Similar to the /download endpoint only it pulls data from build_container_list_v1.json.
 
-# S3 Poller
+# Poller
 
-The S3 Poller lists release manifests (generated by `jenkins/release.rb`
+The Poller lists release manifests (generated by `jenkins/release.rb`
```

Comment:
This sentence is no longer accurate. I believe we need to say something about pulling release data from Bintray.